### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,24 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <!--<div class="stopwatch">
+      <div class="hand second-hand"></div>
+      <div class="hand minute-hand"></div>
+      <div class="hand hour-hand"></div>
+    </div> -->
```
Comment (etojeDenys) [inline src/index.html:16]: remove all comments

File: src/styles/clock.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+@keyframes rotateminute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(6deg);
+  }
+}
```
Comment (etojeDenys) [inline src/styles/clock.scss:72]: why do we need to the same keyframes

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/styles/clock.scss (hunk 2 above)
Comment (maxim2310) [inline src/styles/clock.scss:72]: remove this one and use just rotatesecond everywhere

File: src/styles/clock.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      background-color: $stopwatch-minute-color;
+      animation: rotateminute 5s linear;
+    }
+
+    &--seconds {
+      height: 37vh;
+      width: 2px;
+      top: 1.5vh;
+      background-color: $stopwatch-second-color;
+      animation: rotatesecond 60s linear;
```
Comment (maxim2310) [inline src/styles/clock.scss:51]: and you need to play animation unstoppable

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Great job overall! Just couple tiny things needs to be fixed before approval)

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**